lvmd/lvservice.go

```diff
@@ -45,7 +45,17 @@ func (s *lvService) CreateLV(_ context.Context, req *proto.CreateLVRequest) (*pr
 		return nil, err
 	}
 	oc := s.ocmapper.LvcreateOptionClass(req.LvcreateOptionClass)
-	requested := req.GetSizeGb() << 30
+
+	var requested uint64
```

Comment:
In the proposal, you says that the type of the size should be changed from uint64 to int64 in lvmd, but it seems that you don't do that in this PR. Could you tell me why?
https://github.com/topolvm/topolvm/blob/main/docs/proposals/pvcs-smaller-than-1gi.md#switching-all-occurrences-of-uint64-to-int64-in-lvmd-and-change-size_gb-to-size-in-protobuf-messages

Reply:
You are right that was my initial idea. However, I did notice that if we stick to uint64 precision in lvmd domain, we do not have to touch the lvm_json parsing and casting from int64 to uint64 is safe as long as int64 is positive (which it is by CSI Spec). Thus it requires less code changes to do the conversion here instead of changing lvmd to run with int64. 

If you want, I can change to int64 in all places but it will require more code changes. What is your thought?

Reply:
Thank you for the explanation. That sounds good.